### PR TITLE
Evagelos/replace json with rapidjson

### PR DESCRIPTION
**Pull request by ev-agelos**

### CODE REVIEW

File: spynl/tests/test_responses.py [hunk 1]
```diff
@@ -90,12 +90,9 @@ def test_bad_json(app):
...
     headers = {'Content-Type': 'application/json'}
 
-    with pytest.raises_regexp(AppError,
-                              'Expecting value: line 1 column 1'):
+    with pytest.raises_regexp(AppError, 'Invalid value'):
         app.post('/request_echo', '<a>', headers=headers)
-    with pytest.raises_regexp(AppError,
-                              'Expecting property name enclosed in'
-                              ' double quotes: line 1 column 2'):
+    with pytest.raises_regexp(AppError, 'Missing a name for object member'):
```
Comment (RosanneZe) [inline spynl/tests/test_responses.py:95]: The errors from rapidjson seem to be a bit less descriptive (just an observation, something to keep in mind)

**Review by RosanneZe — APPROVED** [review]
Looks good. I like how you've renamed things.

**Review by o-m-n-i — APPROVED** [review]

**Comment by o-m-n-i** [comment]
@Kareeeeem can you have a look at this PR?

**Review by Kareeeeem — COMMENTED** [review]
Flew under my radar. Approved.

**Pull request merged**